internal/poller/gotimestatus/store.go

```diff
@@ -0,0 +1,84 @@
+package gotimestatus
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/go-redis/redis"
+)
+
+const (
+	redisKey     = "poller:gotime:last_status"
+	redisTestKey = "poller:gotime:test_key"
```

Comment:
@dlsniper should these be changed? I changed the `last_id` to `last_status` but kept the `test_key` the same as the other poller. Maybe a better convention would be to change them both to be prefixed with `poller:gotimestatus:` instead?

Reply:
I agree. Let's do that.

Reply:
Done